### PR TITLE
TK2.8: add services args to observable routes

### PR DESCRIPTION
**Pull request by frenchy64**
Epic: threatgrid/iroh#3986
Extracted from https://github.com/threatgrid/ctia/pull/924

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK2.8: add services args to observable routes
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

**Review by yogsototh — APPROVED** [review]
LGTM!

File: src/ctia/bundle/core.clj [hunk 1]
```diff
@@ -97,14 +97,14 @@
 (s/defn find-by-external-ids
   [import-data entity-type auth-identity
    {{:keys [read-store]} :StoreService
-    :as _services_} :- APIHandlerServices]
+    :as services} :- APIHandlerServices]
```
Comment (ereteog) [inline src/ctia/bundle/core.clj:100]: `services` is not used

File: src/ctia/observable/routes.clj [hunk 2]
```diff
@@ -25,7 +27,10 @@
             (s/optional-key :sort_by)
             (describe (s/enum :id) "Sort result on a field")))
 
-(defroutes observable-routes
+(s/defn observable-routes [{{:keys [get-in-config]} :ConfigService
+                            {:keys [read-store]} :StoreService
+                            :as services} :- APIHandlerServices]
```
Comment (ereteog) [inline src/ctia/observable/routes.clj:32]:
```suggestion
                            {:keys [read-store]} :StoreService} :- APIHandlerServices]
```

**Review by ereteog — APPROVED** [review]
LGTM

File: src/ctia/observable/routes.clj (hunk 2 above)
Comment (frenchy64) [reply src/ctia/observable/routes.clj:32]: Thanks, I'm juggling lots of these `_services_` vs `services` vs nothing. If you'll indulge me, using `_services_` is a little easier for me to manage for (as-of-now) unused service args.

File: test/ctia/entity/web_test.clj [hunk 3]
```diff
@@ -456,6 +457,10 @@
                (is (= 401 (:status (get-judgement jwt-2))))
                (is (= 2 @counter)
                    "Making a call to another JWT should generate another call")
+               (assert (= test-cache-ttl (p/get-in-global-properties
+                                           [:ctia :http :jwt :http-check :cache-ttl]))
+                       (p/get-in-global-properties
+                         [:ctia :http :jwt :http-check :cache-ttl]))
```
Comment (frenchy64) [inline test/ctia/entity/web_test.clj:463]: Sanity check for an observed flaky test. The `(is (= 3 @counter) ..)` failed for some reason.

**Pull request merged**